<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT_StirTrace</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button_StartProcessing">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Processing</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="FolderSelectToolButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>28</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="choose folder"/>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Additive Noise</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Cut Filtering</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_addnoise">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_median">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rlines">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rcolumns">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_XStretch">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_YShear">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_scaling">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cropping">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_banding">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Lines</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Columns</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X Stretching</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Shearing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rescaling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cropping</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_rotate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Banding</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Button_StartProcessing</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotStartProcessing()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FolderSelectToolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFolderBrowser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotStartProcessing()</slot>
  <slot>showFolderBrowser()</slot>
 </slots>
</ui>
